src/python/pants/help/help_formatter.py

```diff
@@ -53,6 +53,13 @@ def add_option(ohis, *, category=None):
             add_option(oshi.advanced, category="advanced")
         if self._show_deprecated:
             add_option(oshi.deprecated, category="deprecated")
+        if oshi.advanced and not self._show_advanced:
+            lines.append(
+                self.maybe_green(
+                    f"There are advanced options which you can list by running "
```

Comment:
Maybe this wording?

> Advanced options available. You can list them by running `{pants_bin()} help-advanced {oshi.scope}`.

Otherwise, annoying grammar nit: you either need to change `which` to be `that`, or use a comma in-between `options` and `which`.

Reply:
I chose your wording, even better!

Reply:
Chose your wording, thank you!